PyRDF/backend/Spark.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import print_function
+from .Dist import Dist
+from pyspark import SparkConf, SparkContext
+
+class Spark(Dist):
+    """
+    Backend that executes the computational graph
+    using using `Spark` framework for distributed
+    execution.
+
+    """
+    def __init__(self, config = {}):
+        """
+        Creates an instance of the Spark
+        backend class.
+
+        Parameters
+        ----------
+        config : dict (optional)
+            The config options for Spark backend. The default
+            value is an empty Python dictionary `{}`. Config
+            specifically reads three options, 'SparkContext'
+            or 'SparkConf' and 'npartitions'. If 'SparkContext'
+            is not given, 'SparkContext' uses 'SparkConf' option,
+            which by default is 'None' and 'npartitions' is
+            '2' by default.
+
+        """
+        super(Spark, self).__init__(config)
+
+        try:
```

Comment:
This `try` `except` can be replaced by just one line: `self.npartitions = config.pop('npartitions', None)`

Reply:
Oh yes. Thanks @etejedor . Will fix this 👍